### PR TITLE
Basic Code Style Check

### PR DESCRIPTION
**Pull request by jorgeorpinel**
With PHP_CodeSniffer and PSR-1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +36,17 @@ CURL used to include a list of accepted CAs, but no longer bundles ANY CA certs.
...
 ```
 
+## How to Test
+
+Unit tests in this SDK can be run using PHPUnit. The test cases are located in the test/Controllers/ dir.
+
+1. Make sure you've installed the dependencies using composer including the `require-dev` dependencies.
+1. Run `vendor/bin/phpunit` from command line to execute the test suite.
+   See https://phpunit.de/manual/current/en/textui.html for info on test output format as well as more command-line options.
+1. Optionally, to check the code standards you may run `vendor/bin/phpcs --standard=PSR1 test`. (No output means no problems.)
```
Comment (jorgeorpinel) [inline README.md]: Moved this block to after **How to Build** and added `phpcs` instructions to lint with PSR-1.

File: test/Controllers/CategoriesTest.php [hunk 2]
```diff
@@ -3,6 +3,8 @@
  * Categories SDK Controller Test Case
  */
 
+namespace GonebusyLib\Tests;
```
Comment (jorgeorpinel) [inline test/Controllers/CategoriesTest.php]: All test classes needed this to pass PSR-1.

**Comment by jorgeorpinel** [comment]
@alexagranov you may want to merge #27 first since this PR also includes the `BookingsTest.php` file... (sry)

This is quite simple, adding a Composer dev dependency, and instructions on the README on how to use it for checking code standards with PSR-1.

I could add a phpcs.xml file so just running `vendor/bin/phpcs` knows what to do, I guess in case you want to add the command to Travis (which I could also try doing). Shall I?

**Comment by jorgeorpinel** [comment]
p.s.

> I could add a phpcs.xml file so just running vendor/bin/phpcs knows what to do...

Actually I did that now and added `phpcs -v` to `.travis`. Please check, Alex.

File: .travis.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@ php:
 install: composer install
 script:
   - vendor/bin/phpunit -v
+  - vendor/bin/phpcs -v
```
Comment (alexagranov) [inline .travis.yml]: Cool, but we should place linters first before the actual tests.

**Comment by alexagranov** [comment]
Need to rebase to master too please.

**Comment by jorgeorpinel** [comment]
All done, @alexagranov (:

**Pull request merged**